src/Http/Session/ArraySessionHandler.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Http\Session;
+
+use SessionHandlerInterface;
+use Tempest\Clock\Clock;
+
+final class ArraySessionHandler implements SessionHandlerInterface
```

Comment:
I'm curious because I've never really looked into session handling. When would an in-memory array session handler without persistence be useful? During testing?

Reply:
Primarily during testing, yeah. 🙂